modules/test/playwright/env/osgi-modules.list

```diff
@@ -1 +1,2 @@
 modules/apps/users-admin/users-admin-demo
+modules/apps/frontend-data-set/frontend-data-set-sample-web
```

Comment:
This is not the right place to add module, it is the global list. It should be in local `env`, sibling to CX list. Like this, all projects would deploy FDS sample module.

Reply:
currently, I added the CX (`workspaces/liferay-sample-workspace/client-extensions/liferay-sample-fds-filter`) to the `tests/client-extension-web`. 

@markocikos Should it instead go in `tests/frontend-data-set-views-web`?

Reply:
CXs should go in `tests/frontend-data-set-views-web/env/client-extensions.list`.
Modules should go in `tests/frontend-data-set-views-web/env/osgi-modules.list`.